### PR TITLE
Restyle Marketplace tab bar

### PR DESCRIPTION
**Pull request by madeincosmos**
### All Submissions:

* [x] Have you followed the [WooCommerce Contributing guideline](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:
Update the Marketplace page menu styles to match the current design.

### How to test the changes in this Pull Request:

1. Check out this branch in local environment
2. Go to **WooCommerce > Marketplace** on your local test site
3. Check there is a white navigation bar under the header, and that it looks good on all screen sizes (mobile CSS coming soon).

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your changes, as applicable?
* [x] Have you successfully run tests with your changes locally?

### FOR PR REVIEWER ONLY:

* [ ] I have reviewed that everything is sanitized/escaped appropriately for any SQL or XSS injection possibilities. I made sure Linting is not ignored or disabled.

### CODE REVIEW

**Comment by madeincosmos** [comment]
Added mobile styles for the top bar so that it's scrollable

File: assets/css/admin.scss [hunk 1]
```diff
@@ -81,6 +80,59 @@
...
+				font-size: 14px;
+				line-height: 20px;
+				margin: 0;
+			}
+
+			a,
+			a:visited,
+			a:hover,
+			a:focus {
+				border: none;
```
Comment (andfinally) [inline assets/css/admin.scss:118]:
```suggestion
				border: none;
				box-shadow: none;
```

To get rid of this:

image

**Review by kloon — COMMENTED** [review]
Changes is looking good and working, just a specific range between 783 and 1066px where the menu is pushed to a new line and not scrollable.

**Comment by madeincosmos** [comment]
Thank you @kloon, I made the menu scrollable on medium-sized screens as well. Also centered it on larger screens so that it's in line with @andfinally's changes in #30478:

**Comment by andfinally** [comment]
I was just puzzling over the medium-sized screens... The current fix should do for now, but I think this is going to become an issue when we add more categories, and in languages with longer words. I'll mention this to the designers on the project thread.

File: includes/admin/views/html-admin-page-addons.php [hunk 2]
```diff
@@ -33,7 +33,19 @@
...
 		</form>
 	</div>
-
+	<div class="top-bar">
+			<ul>
+				<?php foreach ( $sections as $section ) : ?>
+					<li>
+						<a
+							class="<?php echo $current_section === $section->slug ? 'current' : ''; ?>"
+							href="<?php echo esc_url( admin_url( 'admin.php?page=wc-addons&section=' . esc_attr( $section->slug ) ) ); ?>">
```
Comment (andfinally) [inline includes/admin/views/html-admin-page-addons.php:43]: I wonder if we need the `esc_attr`, given we're escaping the whole URL?

File: assets/css/admin.scss [hunk 3]
```diff
@@ -81,6 +80,56 @@
...
+				color: #1e1e1e;
+				display: inline-block;
+				text-decoration: none;
+				outline: none;
+				padding: 20px 16px;
```
Comment (andfinally) [inline assets/css/admin.scss]: As a temporary measure, to let the bar fit on 1024:

```suggestion
				padding: 20px 12px;
```

**Review by andfinally — APPROVED** [review]
Looks lovely, thank you! I just left a suggestion for a temporary workaround to make the items narrower, till we can sort out the wrapping.

**Review by kloon — APPROVED** [review]
Perfect, thanks for those changes, looks much better and ready from my side.

File: includes/admin/views/html-admin-page-addons.php (hunk 2 above)
Comment (madeincosmos) [reply includes/admin/views/html-admin-page-addons.php:43]: This is how it's working currently in latest trunk, so I'd probably leave it as it is:

https://github.com/woocommerce/woocommerce/blob/a54df37363c881ef1ba855c5fce470caf67e7633/includes/admin/views/html-admin-page-addons.php#L35

File: assets/css/admin.scss (hunk 1 above)
Comment (madeincosmos) [reply assets/css/admin.scss:118]: Thank you @andfinally, fixed it now.

File: assets/css/admin.scss (hunk 3 above)
Comment (madeincosmos) [reply assets/css/admin.scss]: Fixed it.

**Comment by andfinally** [comment]
The designers have proposed a popover for these tabs – I've created issue 11009-gh-Automattic/woocommerce.com for that, but in the meantime we can merge this PR – it works for now.

**Comment by rodelgc** [comment]
I will remove the `status: needs testing instructions` label because the testing for this PR will be covered in [Marketplace mobile layout fixes](https://href.li/?https://github.com/woocommerce/woocommerce/pull/30498) and [Add mobile section menu to marketplace](https://href.li/?https://github.com/woocommerce/woocommerce/pull/30499). Reference: p6q8Tx-27P-p2#comment-5577

**Pull request merged**